Biblioteca.h

```diff
@@ -0,0 +1,89 @@
+#pragma once
+#include"ArrayGenerico.h"
+#include"Alumno.h"
+#include"Libro.h"
+class Biblioteca {
+
+private:
+
+	string nombreBiblioteca;
+	
+	ArrayGenerico<Alumno> alumnos;
+	ArrayGenerico<Libro> libros;
+
+public:
+
+	Biblioteca(string nombreBiblioteca, int capaciadadAlumnos, int capacidadLibros) : alumnos(capaciadadAlumnos), libros(capacidadLibros) {
+
+		this->nombreBiblioteca = nombreBiblioteca;
+	}
+
+	void agregarAlumno(Alumno* nuevoAlumno) {
+		alumnos.agregar(nuevoAlumno);
+	}
+
+	void agregarLibro(Libro* nuevoLibro) {
+		libros.agregar(nuevoLibro);
+	}
+
+	Alumno* buscarAlumno(int ci) {
+		for (int i = 0; i < alumnos.getCantidad(); i++) {
+			if (alumnos.getElemento(i)->getCi() == ci) {
+				return alumnos.getElemento(i);
+			}
+		}
+		throw ExcepcionAlumnoNoExiste(ci);
+	}
+
+	Libro* buscarLibro(int codigo) {
+		for (int i = 0; i < libros.getCantidad(); i++) {
+			if (libros.getElemento(i)->getCodigo() == codigo) {
+				return libros.getElemento(i);
+			}
+		}
+		throw ExcepcionLibroNoExiste(codigo);
+	}
+
+	//REGISTRAR PRESTAMOS
+
+	void registrarPrestamo(int ciAlumno, int codigoLibro) {
+
+		Alumno* alumnito = buscarAlumno(ciAlumno);
+		Libro* librito = buscarLibro(codigoLibro);
+
+		if (alumnito->getLibro() != nullptr)
+			throw ExcepcionAlumnoLibro(alumnito->getNombre());

```

Comment:
Primero, Mejorar nombre "AlumnoLibro" de la excepcion, por is solo no se entiende el porque..
Segundo, por standart, la palabra excepcion deberia ir al final
```suggestion
			throw AlumnoLibroExcepcion(alumnito->getNombre());

```